### PR TITLE
Named some tokens and method

### PR DESCRIPTION
**Pull request by DoktorSAS**
This is from a new and update fork of the project.

### CODE REVIEW

File: src/client/game/scripting/function_tables.cpp [hunk 1]
```diff
@@ -2984,7 +2984,7 @@ namespace scripting
 		{"primaryweapon", 0x33D},
```
Comment (mjkzy) [inline src/client/game/scripting/function_tables.cpp:2984]: need to call this `_not_primaryweapon` if ur gonna have another token that's the exact same name

File: src/client/game/scripting/function_tables.cpp [hunk 2]
```diff
@@ -3534,73 +3565,212 @@ namespace scripting
...
+		{"rankTable", 0x713a},
+		{"recoilScale", 0x7268},
+		{"regenSpeed", 0x72d4},
+		{"registerHalfTimeDvar", 0x72ef },
+		{"registerNumLivesDvar", 0x72f4 },
+		{"registerRoundLimitDvar", 0x72f6 },
+		{"registerRoundSwitchDvar", 0x72f7 },
+		{"registerScoreLimitDvar", 0x72f8 },
+		{"registerTimeLimitDvar", 0x72f9 },
+		{"registerWinLimitDvar", 0x72fe },
```
Comment (mjkzy) [inline src/client/game/scripting/function_tables.cpp:3668]: fix the space between the offsets and }

**Review by mjkzy — CHANGES_REQUESTED** [review]

File: src/client/game/scripting/function_tables.cpp (hunk 2 above)
Comment (DoktorSAS) [reply src/client/game/scripting/function_tables.cpp]: Solved.

File: src/client/game/scripting/function_tables.cpp [hunk 3]
```diff
@@ -3534,73 +3565,212 @@ namespace scripting
 		{"getTeamSpawnPoints", 0x411f},
 		{"getWeaponClass", 0x4167},
 		{"giveLoadout", 0x41e0}, // this may not even be giveLoadout but it's a wrapper for it and it does the same logic so
+		{"gotPullbackNotify ", 0x428b},
```
[the remark this answers is not in the document]
Comment (DoktorSAS) [reply src/client/game/scripting/function_tables.cpp]: Solved

File: src/client/game/scripting/function_tables.cpp (hunk 1 above)
Comment (DoktorSAS) [reply src/client/game/scripting/function_tables.cpp]: I forgot about it, solved.

**Pull request merged**